hexa/plugins/connector_dhis2/datacards.py

```diff
@@ -33,6 +57,8 @@ class InstanceCard(Datacard):
 
     external = InstanceSection()
     metadata = OpenHexaMetaDataSection(value="index")
+    # FIXME: this should be shown only if the user has permission to see the credentials
+    usage = UsageSection()
```

Comment:
Let's fix it! We already have a similar use case with datacard `Action`, using the `enabled_when` parameter (see https://github.com/BLSQ/openhexa-app/blob/main/hexa/ui/datacard/actions.py#L25). This could be easily replicated for this use case.